test262/main.js

```diff
@@ -118,6 +136,50 @@
       }
     }
 
+    const performanceChartStartDateTime = DateTime.fromISO("2021-07-04");
+
+    for (const entry of data) {
```

Comment:
Can we avoid iterating over this again? We should be able to easily merge this with the `for (const entry of data)` above, no?